src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/DotNetCliToolReference.xaml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule
+    Name="DotNetCliToolReference"
+    DisplayName="Package"
+    PageTemplate="generic"
+    Description="Package"
+    xmlns="http://schemas.microsoft.com/build/2009/properties">
+    
+    <Rule.DataSource>
+        <DataSource Persistence="ProjectFile" ItemType="DotNetCliToolReference" HasConfigurationCondition="False" />
+    </Rule.DataSource>
+    
+    <StringProperty Name="Description" 
+                    ReadOnly="True" 
+                    Visible="True"
+                    DisplayName="Description" 
+                    Description="Dependency description." />
+
+    <StringProperty Name="Version"
+                    ReadOnly="True"
+                    DisplayName="Version"
+                    Description="Version of dependency.">
+    </StringProperty>
+
+    <StringProperty Name="Name" 
```

Comment:
I'm just mirroring what is in PackageReference, since that is what it is. I doubt all the properties are needed in either - most return empty strings.